packages/devtools-reps/src/reps/error.js

```diff
@@ -82,7 +82,16 @@ function ErrorRep(props) {
  */
 function getStacktraceElements(props, preview) {
   const stack = [];
-  preview.stack
+  if (!preview.stack) {
+    return stack;
+  }
+
+  const isStacktraceALongString = preview.stack.type === "longString";
```

Comment:
Could we somehow utilize `isLongString()` from reps.js?

Reply:
sure